### DISCUSSION TITLE
Farmer's Delight + Aquaculture 2

### DISCUSSION THREAD

**Discussion by skywol1ker** [vectorwing/FarmersDelight#445] [Ideas]
Hey!  I fell in love with Farmer's Delight! Thanks to him, I rediscovered minecraft mods. And looking for new mods (food related), I found Aquaculture 2. He is responsible for adding new fish to the game. Everything related to the fish themselves is done very cool! But the additional content is done, in my opinion, rather poorly.

And then it dawned on me! After all, Farmer’s Delight doesn’t touch on the topic of fish at all! And here both mods can complement each other! But Aquaculture 2 isn't perfect and in my opinion here's what needs to be done to make both mods play well with each other:
 1. Remove: Neptunium ingot and everything related to it;
 2. Keep: fish, new rods, new hooks, worm farm block, tackle box block, new items (which drop when fishing);
 3. Replace: knives, boards, fish fillets, craft food (Integration with "Farmer's Delight");
 4. Add: new food related to fish.

Yes, there is a "Farmers's Delight Compats" mod, but:
 1. This is the usual integration of existing recipes and knives (no significant changes);
 2. Version 1.16.5 and below only.

I would do it myself! But I'm only delving into the topic of mods and I don't know how to edit them at all ... Can you recommend resources for studying this topic? And if someone who knows how to work with existing mods is reading this: how do you like the idea?

In general, this is all I wanted to write!  Thank you for your attention ❤️

**Comment by artheeist** [comment]
Hello! I love this idea and have been looking for something of this sort for a while now. I have very basic experience in modding (made some very simple stuff many years ago), but I would love to help you somehow, if you're really getting into this. I believe MrCrayfish had a video series on youtube about creating mods, and I can tell you from personal experience that reverse engineering existing mods is a great way to learn how things work.
